website/util/__init__.py

```diff
@@ -123,7 +123,7 @@ def api_v2_url(path_str,
     return str(base_url)
 
 
-def web_url_for(view_name, _absolute=False, _guid=False, *args, **kwargs):
+def web_url_for(view_name, _absolute=False, _internal=False, _guid=False, *args, **kwargs):
```

Comment:
I see we have a couple duplicative `web_url_for` methods, do these need to be vetted?